app/javascript/packages/analytics/index.js

```diff
@@ -0,0 +1,20 @@
+import { loadPolyfills } from '@18f/identity-polyfill';
+
+const LOGGER_ENDPOINT = '/api/logger';
```

Comment:
Admittedly not ideal to hard-code this here.

Couple alternatives I considered:

- Some "global" variables bootstrapped into the base page layouts
- Initialize this from each consumer, and handle "bootstrapping" as part of that consumer's setup
- Something else where `javascript_pack_tags_once` knows that this is being loaded and "tags on" some extra initialization data when added to the page

Reply:
I like the idea of bootstrapping via base page `data-` tags

Reply:
> I like the idea of bootstrapping via base page `data-` tags

Bootstrapping in 615abad.